Integrations/integration-IntSights.yml

```diff
@@ -1026,6 +1051,30 @@ script:
     - contextPath: IntSights.Alerts.IsFlagged
       description: Whether or not the alert is flagged
       type: boolean
+    - contextPath: IntSights.Alerts.Tags.CreatedBy
+      description: Name of the service the tag was created by
+      type: string
+    - contextPath: IntSights.Alerts.Tags.Name
```

Comment:
We should use singular in context output, so let's use `Tag` instead of `Tags`

Reply:
But it can contain multiple tags, this make no sense of making it singular